src/materialized/src/lib.rs

```diff
@@ -295,7 +295,7 @@ async fn serve_stash<S: mz_stash::Append + 'static>(
     let storage_service = orchestrator
         .namespace("storage")
         .ensure_service(
-            "runtime",
+            "storaged",
```

Comment:
What's the rationale for changing this? `runtime` was intentional because the orchestrator will prefix the service name with the namespace name, so `storage-runtime`.